.github/workflows/coverage.yml

```diff
@@ -4,8 +4,6 @@ on:
   push:
     branches:
       - v0.34.x
-      - main
```

Comment:
We don't need to run v0.34.x workflows on the `main` branch - the `main` branch has its own workflows.